nestjs/src/opportunities/opportunities.controller.ts

```diff
@@ -22,7 +24,11 @@ import { CurrentRequest, DefaultGuard, RequiredRoles, Role, RoleGuard } from 'sr
 import { ApplicantResumeDto } from './dto/applicant-resume.dto';
 import { ConsentDto } from './dto/consent.dto';
 import { ResumeDto } from './dto/resume.dto';
+import { StatusDto } from './dto/status.dto';
+import { VisibilityDto } from './dto/visibility.dto';
+//import { FormDataDto } from './dto/form-data.dto';
```

Comment:
is it necessary or left after the migration?

Reply:
nope, this should be deleted, thx